pkgs/tools/typesetting/tex/texlive/fixHashes.awk

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env gawk -f
```

Comment:
By default, passing args to something under `/usr/bin/env` doesn't work.  On my machine it gives this:

```console
$ nix-build ../../../../.. -Q --no-out-link -A texlive.scheme-full.pkgs | ./fixHashes.awk > ./fixedHashes-new.nix
/usr/bin/env: ‘gawk -f’: No such file or directory
/usr/bin/env: use -[v]S to pass options in shebang lines
```

If we make this change, it works provided the user has a new enough coreutils, which NixOS does:

```suggestion
#!/usr/bin/env -S gawk -f
```

https://www.gnu.org/software/coreutils/manual/html_node/env-invocation.html#g_t_002dS_002f_002d_002dsplit_002dstring-usage-in-scripts

Reply:
@bhipple Nice catch. I was, actually, not completely happy with the shebang because gawk may not be installed in the system. The updated version uses nix-shell in shebang to address both of the issues.